recipes/rhapsody/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "0.2.1" %}
+{% set name = "rhapsody" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/r/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d73e0af9f57fdb7e48bd7bab4f946fbeb0240f6d44cb47e0257a277df7051beb
+
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+  entry_points:
+    - qiime2.plugins = rhapsody.q2.plugin_setup:plugin
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - numpy >=1.13.1
```

Comment:
Is `numpy` needed here? It's not obvious from the `setup.py` that it is
```suggestion
```

Reply:
This is actually necessary, because both tensorflow and scikit-bio builds will fail if numpy < 1.13 were installed.

Reply:
Then tensorflow and scikit-bio should be pinned appropriately. Nominally you shouldn't have to worry about the dependencies of your dependencies.

Regardless they're not being installed in the `host` environment so the numpy installation here isn't being used for anything.

Reply:
@chrisburr good point.  Dependencies have been trimmed.